### PR TITLE
Redownload subs when clicking in the flag

### PR DESCRIPTION
**Pull request by fernandog**
image

### CODE REVIEW

**Comment by fernandog** [comment]
using @Elettronik idea from SickRage

File: gui/slick/js/ajax-episode-subtitles.js [hunk 1]
```diff
@@ -63,3 +63,62 @@
...
+                } else {
+                    img.prop('src', 'images/' + yesImage);
+                    disableLink(link);
+                }
+
+                // put the corresponding image as the result of queuing of the manual search
+                img.prop('title', imageResult);
+                img.prop('alt', imageResult);
+                img.prop('height', options.size);
+                img.prop('src', 'images/' + imageName);
```
Comment (fernandog) [inline gui/slick/js/ajax-episode-subtitles.js]: @OmgImAlexis i think all this can go away right?

Comment (OmgImAlexis) [inline gui/slick/js/ajax-episode-subtitles.js]: That looks right, if it works then that's good. I'll fix it all up at a later date.

File: gui/slick/js/ajax-episode-subtitles.js [hunk 2]
```diff
@@ -66,8 +66,8 @@
 
 (function() {
     $.ajaxEpRedownloadSubtitle = function() {
-        $('.epRedownloadSubtitle').on('click', function() {
-            event.preventDefault();
+        $('.epRedownloadSubtitle').on('click', function(evt) {
+            evt.preventDefault();
```
Comment (fernandog) [inline gui/slick/js/ajax-episode-subtitles.js]: @OmgImAlexis why I need to add this?

Comment (OmgImAlexis) [inline gui/slick/js/ajax-episode-subtitles.js]: We name it `e` for consistency, that's to prevent the link actually going to the url.

**Comment by fernandog** [comment]
@OmgImAlexis can you review this? its ready

File: gui/slick/js/ajax-episode-subtitles.js [hunk 3]
```diff
@@ -66,8 +66,8 @@
 
 (function() {
     $.ajaxEpRedownloadSubtitle = function() {
-        $('.epRedownloadSubtitle').on('click', function(evt) {
-            evt.preventDefault();
+        $('.epRedownloadSubtitle').on('click', function(e) {
+            e.preventDefault();
```
Comment (fernandog) [inline gui/slick/js/ajax-episode-subtitles.js]: @OmgImAlexis like this?

**Comment by fernandog** [comment]
Added in the manual subtitle search

**Pull request closed without merging**